### DISCUSSION TITLE
太喜欢这个组件库了！

### DISCUSSION THREAD

**Discussion by lanbinshijie** [hellof2e/quark-design#216] [General]
今天早上我在公众号文章上看到了这个UI组件库的介绍。当时我还没太在意，因为公众号文章没有给渲染图。

在我看来，大多数的组件库局限性都太强了。而且很少能见到一个专门有为手机制作适配的组件库。所以当时我并没有太在意。

当我点开这个组建库的预览界面时，我被深深的震撼到了。没错，这个组件库就是我想要的那个样子。瞬间感觉手里的element UI，naiveUI都不香了。

特别支持这个组件库，希望它可以继续壮大发展！开发团队加油！

**Comment by lanbinshijie** [comment]
追评：
当体验完所有的组件并上手制作了一个demo之后，我深深的体验到了这个组件库的强大。

这个组件库上手十分简单，同时，多样的组件，不同的排列组合就能组建出焕然一新的界面，而不用担心千篇一律的问题。

特别是现在行业发展逐渐的往移动端和小程序上靠拢，这样的组件库就成为了至关重要的必需品。

这样的组件库可以让前端工程师在前端转小程序和APP开发之前有一个过渡。

最后我十分支持这样组建库的开发，希望quark-design可以继续加油，开发出更多有用的组件。

**Reply by iamagentleboy** [reply]
quark-design 中如何更改filed的input的style

**Reply by xxxsf** [reply]
目前支持以下修改：

![image](https://github.com/hellof2e/quark-design/assets/14307551/aed06687-c518-4ef4-b9be-23081aeb0bf7)


文档底部：https://vue-quarkdesign.hellobike.com/#/zh-CN/component/field

如果还不满足你要改的，可以说一下，具体指哪个style？

**Reply by xxxsf** [reply]
参考主流组件库设计，quarkd 没有边框和radius，能否截图看下您的设计稿？看看我们是否要去支持

**Reply by xxxsf** [reply]
这样不就行了吗？不用通过css变量修改。
![image](https://github.com/hellof2e/quark-design/assets/14307551/9d40c2c6-697e-4568-b5fc-1314fb72602e)

然后隐藏 label
![image](https://github.com/hellof2e/quark-design/assets/14307551/b8004004-830f-464d-9378-91ce123ec680)

**Comment by lanbinshijie** [comment]
以上所有的文字仅属于个人的一点点拙见，欢迎各位大佬批评指正。

**Reply by xxxsf** [reply]
感谢🙏

**Comment by iamagentleboy** [comment]


**Reply by xxxsf** [reply]
那就需要一个单独的input组件，我回头补充一个进去，到时候发你链接好了

**Comment by yg7001** [comment]
只能用于移动端吗？

**Comment by xxxsf** [comment]
@yg7001   
quarkd （移动端组件库）用于移动端：https://github.com/hellof2e/quark-design
quarkc （构建你自己的业务组件）可以用于任何PC/web/移动端：https://github.com/hellof2e/quark-core

**Comment by ZinLiao** [comment]
服务组件如何实现？
